daemon/cmd/kube_proxy_replacement.go

```diff
@@ -41,12 +41,12 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-func initKubeProxyReplacementOptions() (strict bool) {
+func initKubeProxyReplacementOptions() (bool, error) {
```

Comment:
hint to leave a comment about the returned values since now we don't know what is `bool` used for.

Reply:
@aanm I updated this function with a comment describing its usage, can you please read this comment and ensure I have it's usage summarized correctly.